CMakeLists.txt

```diff
@@ -705,6 +799,13 @@ set(test_helper_sources
 	t/helper/test-trace2.c t/helper/test-urlmatch-normalization.c t/helper/test-xml-encode.c 
 	t/helper/test-wildmatch.c t/helper/test-windows-named-pipe.c t/helper/test-write-cache.c)
 
+#https://github.com/git-for-windows/git does not have test-advice.c but replaces it with test-cmp.c and test-iconv.c
+if(EXISTS ${CMAKE_SOURCE_DIR}/t/helper/test-advise.c)
+	list(APPEND test_helper_sources t/helper/test-advise.c)
```

Comment:
We don't have `test-advise`? Indeed. But that's because we're based on top of v2.26.1, still, and core Git's `master` has `test-advise`. Once Git for Windows will be rebased on top of v2.27.0 (i.e. as soon as that version comes out), we will want to have it here, too.

In other words, that `test-advise.c` should not be in this patch series, not as long as we're targeting Git for Windows' `master` (otherwise this won't work on Linux or macOS, which Git for Windows very much wants to support, too, e.g. for Scalar).

Reply:
So, you are saying test-advice.c will be present in git-for-windows 2.27? Okay. 
So, you are saying to modify the test source with respect this fork. When sending to git/git we have to modify it? Is that right?